### PR TITLE
[pcysys-771] Bugfix: Date in Full action report was changed, caused a parsing exce…

### PR DESCRIPTION
**Pull request by TalPcysys**
…ption

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Pcysys/Integrations/Pcysys/Pcysys.py [hunk 1]
```diff
@@ -187,9 +187,10 @@ def _parse_date(full_date, separator):
 
         res_list: List[dict] = []
         for ordered_dict in full_action_report_list:
-            full_date_to_convert = ordered_dict['Time'].replace(' ', '')
-            full_date_list = full_date_to_convert.split(',')
-            day, month, year = _parse_date(full_date_list[0], '/')
+            full_date_to_convert = ordered_dict['Time']
+            LOG(f'full_date_to_convert: {full_date_to_convert}')
```
Comment (ronykoz) [inline Packs/Pcysys/Integrations/Pcysys/Pcysys.py]: Do we need this message here?

File: Packs/Pcysys/Integrations/Pcysys/Pcysys.py [hunk 2]
```diff
@@ -187,9 +187,10 @@ def _parse_date(full_date, separator):
...
+            full_date_list = full_date_to_convert.split(' ')
+            year, month, day = _parse_date(full_date_list[0], '-')
```
Comment (ronykoz) [inline Packs/Pcysys/Integrations/Pcysys/Pcysys.py]: maybe you would like to use library like dateutil so the format will not hurt you as much in the future?

Comment (TalPcysys) [reply Packs/Pcysys/Integrations/Pcysys/Pcysys.py]: For sure, it is only a temporary fix for this version. In the next release I'll wrap it up with a library

File: Packs/Pcysys/Integrations/Pcysys/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
-## [Unreleased]
+## [20.03.10] - 2020-03-10
+Fixed an issue date parsing
```
Comment (ronykoz) [inline Packs/Pcysys/Integrations/Pcysys/CHANGELOG.md]:
```suggestion
Fixed an issue with date parsing for the `pentera-get-task-run-full-action-report` command.
```

**Comment by ronykoz** [comment]
Just update the Chnagelog message and we are good to go.

**Comment by ronykoz** [comment]
@TalPcysys Seems that you have a conflict with the CHANGELOG file and i'm unable to fix it myself, due to permissions issue.

**Comment by ronykoz** [comment]
@TalPcysys the UT are also failing, please have a look at that

File: Packs/Pcysys/Integrations/Pcysys/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+Fixed an issue with date parsing for the 
+`pentera-get-task-run-full-action-report` command.
```
Comment (ronykoz) [inline Packs/Pcysys/Integrations/Pcysys/CHANGELOG.md]:
```suggestion
Fixed an issue with date parsing for the `pentera-get-task-run-full-action-report` command.
```

**Pull request merged**